### PR TITLE
Added an example notebook

### LINKED ISSUE #9: Demo a use case with simple example
Create a use case demo by taking a simple example dataset and using the encryption technique on it.

### PR DESCRIPTION
**Pull request by suppathak**
In this PR, we add a simple example notebook in order to understand a basic underlying principle behind homomorphic encryption. 

Closes #9

### CODE REVIEW

**Comment by suppathak** [comment]
Installed reviewnb,
Review it here : https://app.reviewnb.com/redhat-et/homomorphic-learning/pull/10/

File: notebooks/demo-simple.ipynb [hunk 1]
```diff
@@ -0,0 +1,264 @@
+{
```
Comment (aakankshaduggal) [inline notebooks/demo-simple.ipynb:1]: <p>nit - encrypted*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/homomorphic-learning/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Review by aakankshaduggal — COMMENTED** [review]
Overall looks good, simple, and easy-to-understand notebook.
However, we don't need the checkpoint notebook to be added. Can you please remove that and we will be all set to ship it.

**Comment by suppathak** [comment]
Thanks @aakankshaduggal for the feedback. I have updated the changes.

**Pull request merged**